contracts/marketplace/src/testing/setup/setup_accounts.rs

```diff
@@ -1,10 +1,61 @@
 use crate::error::ContractError;
-use crate::testing::setup::constants::{CREATION_FEE, SECOND_BIDDER_INITIAL_BALANCE};
 use cosmwasm_std::{coins, Addr, Coin};
-use cw_multi_test::{BankSudo, SudoMsg as CwSudoMsg};
+use cw_multi_test::SudoMsg as CwSudoMsg;
+use cw_multi_test::{BankSudo, SudoMsg};
 use sg_multi_test::StargazeApp;
 use sg_std::NATIVE_DENOM;
 
+pub const INITIAL_BALANCE: u128 = 5000000000;
+pub const CREATION_FEE: u128 = 5_000_000_000;
+pub const MINT_PRICE: u128 = 100_000_000;
+pub const SECOND_BIDDER_INITIAL_BALANCE: u128 = 2000;
```

Comment:
I'm confused with this one is this amount `ustars` or `STARS` the ones above seems to be in micro. 

Maybe add a comment on what's the amount being represented by the constant (in STARS)

Reply:
Its in ustars, added comment